packages/cli/src/PublicApi/v1/handlers/audit/audit.handler.ts

```diff
@@ -1,14 +1,15 @@
 import { authorize } from '@/PublicApi/v1/shared/middlewares/global.middleware';
-import { audit } from '@/audit';
+import { SecurityAuditService } from '@/security-audit/SecurityAudit.service';
 import type { Response } from 'express';
 import type { AuditRequest } from '@/PublicApi/types';
+import Container from 'typedi';
 
 export = {
 	generateAudit: [
 		authorize(['owner']),
 		async (req: AuditRequest.Generate, res: Response): Promise<Response> => {
 			try {
-				const result = await audit(
+				const result = await Container.get(SecurityAuditService).run(
```

Comment:
You could make this a dynamic import to make this code truly lazy-loaded.
```suggestion
				const { SecurityAuditService } = await import('@/security-audit/SecurityAudit.service');
				const result = await Container.get(SecurityAuditService).run(
```